### DISCUSSION TITLE
When will NativeBase support Expo SDK 45 or 46.

### DISCUSSION THREAD

**Discussion by jeffthompson1971** [GeekyAnts/NativeBase#5603] [Feature request]
Currently only SDK 44 is supported per my understanding.  I can't even use Expo go anymore because Expo SDK 44 is too old.  When will you support 45/46?

**Comment by ankit-tailor** [comment]
Hey @jeffthompson1971,  SDK 46 is already supported by NativeBase. Are you facing any issues while upgrading?

**Reply by jeffthompson1971** [reply]
Oh is that the case?  I was just always checking your 'next' documentation and I would run your command to create a new expo/native-base project and then checking the package.json and even today it's at expo 44, so I assumed you were not ready yet.    I will look to upgrade my project today.   Thank you so much for getting back to me so quickly.  

Maybe you could update the example 'starter' project to pull in the latest/greatest?  

This is the link I used https://docs.nativebase.io/next/install-expo
And it will result in the following new project:
"dependencies": {
    "expo": "^44.0.0",
    "expo-status-bar": "~1.2.0",
    "native-base": "3.4.0",

**Comment by jeffthompson1971** [comment]
Actually I tried to upgrade but WEB is broken.. it just crashes immediately when I hit 'w' 

Error: Cannot find module '@expo/dev-server/build/webpack/symbolicateMiddleware'
Require stack:
- /Users/jeffreythompson/workspace/mlss-client/node_modules/@expo/cli/build/src/start/server/webpack/WebpackBundlerDevServer.js
- /Users/jeffreythompson/workspace/mlss-client/node_modules/@expo/cli/build/src/start/server/DevServerManager.js
- /Users/jeffreythompson/workspace/mlss-client/node_modules/@expo/cli/build/src/start/startAsync.js
- /Users/jeffreythompson/workspace/mlss-client/node_modules/@expo/cli/build/src/start/index.js
- /Users/jeffreythompson/workspace/mlss-client/node_modules/@expo/cli/build/bin/cli
Error: Cannot find module '@expo/dev-server/build/webpack/symbolicateMiddleware'

**Comment by jeffthompson1971** [comment]
Has anyone been able to do an expo web build with Expo SDK 45 or 46 with native-base?   I've been trying for a couple weeks now and nothing is working.   I was able to scaffold a clean expo 47 project, but I"m not sure if I can use native-base with it.  If not, when will native-base be able to support expo 47?

**Comment by jeltehomminga** [comment]
also a bit confused about the starter template. Does it mean we should not use expo SDK 47 yet? Or is the starter only out of date 🤔

**Comment by bnussman** [comment]
I haven't looked into your exact issue, but Native Base works (and has worked) on the latest Expo SDK. Here is my repo that uses Expo SDK 47. Web and native both work. https://github.com/bnussman/beep

**Reply by Andres6936** [reply]
Same here, we have an application working with NativeBase and Expo 47, exporting for Web, Android and iOS.
